bentoml/handlers/tensorflow_tensor_handler.py

```diff
@@ -90,10 +90,9 @@ def _handle_raw_str(self, raw_str, output_format, func):
             raise NotImplementedError("column format 'inputs' is not implemented")
 
         if output_format == "json":
-            result_object = {"predictions": result}
```

Comment:
@hrmthw removed the leading `{"predictions": ...}` here just for consistency, lmk if you think this will cause other problems

Reply:
It's for consistency of tensor-flow serving only. Removing it will not cause other problems.